docs/contributing.rst

```diff
@@ -370,67 +370,156 @@ Building the documentation
 
 Building the documentation requires installing sphinx::
 
-    pip3 install sphinx
+   pip3 install sphinx
 
-To build the documentation, you need to be in the ``docs`` folder::
+To build the documentation, you need to be in the main ``geomstats`` folder. You can do this with::
 
-    cd docs
+   sphinx-build docs/ docs/html
 
-In the vast majority of cases, you only need to generate the full web site::
 
-    sphinx-build -b html . build/html
-    make html
+Writing Docstrings
+==================
 
+Intro to Docstrings
+^^^^^^^^^^^^^^^^^^^
 
-Guidelines for writing documentation
-====================================
+A docstring is a well-formatted description of your function/class/module which includes 
+its purpose, usage, and other information. 
+
+There are different markdown languages used for docstrings in Python. The most common 
+three are reStructuredText, numpy, and google docstring styles. For geomstats, we are
+using the numpy docstring standard. 
+Check out `NumPy template <https://numpydoc.readthedocs.io/en/latest/format.html>`_ for a thorough walkthrough.
+Following this syntax is important not only for readability, it is also required for automated parsing for inclusion into our generated API Reference.
+
+In Python, each module, class, and function has a ``__doc__`` attribute. It is set by default whenever you include
```

Comment:
I'll remove describing what the doc attribute is--that's fair. I think it'd be fine to keep the example so that people can see how it renders/where it goes.

Reply:
Let me know if you disagree and we can chat.